tox.ini

```diff
@@ -68,21 +68,20 @@ deps =
     online: astroquery
     hypothesis: astroquery
     # Figure tests need a tightly controlled environment
-    figure-!devdeps: astropy==5.1.0
-    figure-!devdeps: matplotlib==3.5.2
+    figure-!devdeps: astropy==5.2.0
+    figure-!devdeps: matplotlib==3.6.2
     figure-!devdeps: mpl-animators==1.0.0
-    figure-!devdeps: numpy<1.24
 extras =
     all
     tests
 commands =
-    pip freeze --all --no-input
+    !figure: pip freeze --all --no-input
```

Comment:
I think it's still worth printing this to the log (as well as piping it ot a file later) - I've found it useful recently.

Reply:
We don't need this as the commands below also cat the file into the logs.
This avoids a repeat of this output.